test/modules/load-gen/docker/linux/arm32v7/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG base_tag=1.0.5-linux-arm32v7
+ARG base_tag=1.0.6.0-linux-arm32v7
 FROM azureiotedge/azureiotedge-module-base:${base_tag}
```

Comment:
Please confirm with Varun and Damon if it is ok to move all base images in EdgeBuilds CR; if so, create another PR.  thanks.